<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleAlarmWidget</class>
 <widget class="QWidget" name="VehicleAlarmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="VA_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="VA_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>400</width>
     <height>120</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(179, 179, 179);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
